.github/workflows/ci.yml

```diff
@@ -3,6 +3,7 @@ name: ⛷ PR tests
 env:
   CI: true
   NODE_VERSION: 18
+  NEXT_PUBLIC_API_BASE_URL: https://staging.dmp.ovio.org
```

Comment:
@sifelix this doesn't seem to be enough for the build to target the staging API. Could you have a quick look?

Reply:
Did you make it work?

Reply:
> env | grep -e NEXT_PUBLIC_ > .env.production
> ./gen_env.sh production

if not something like that should work

Reply:
Thanks @sifelix, I managed to make it work updating .env.staging and loading this instead to stay consistent.

Could you open the CORS on the staging backend? For example, authorize URLs starting with `https://wfp-dmp-`. Thoughts?